errors.go

```diff
@@ -148,12 +148,9 @@ type stackTracer interface {
 
 func defaultErrorHandler(status int, origErr error, c Context) error {
 	env := c.Value("env")
-	ct := defaults.String(httpx.ContentType(c.Request()), defErrorCT)
-	if strings.Contains(ct, "form") {
-		ct = defErrorCT
-	}
-	c.Response().Header().Set("content-type", ct)
+	rct := defaults.String(httpx.ContentType(c.Request()), defErrorCT)
 
+	c.Response().Header().Set("content-type", defErrorCT)
```

Comment:
It looks like the production handler just returns a static template, so utf8 may be fine here as well. Not picky about this, just trying to avoid additional code churn to fix this specific bug:
```func productionErrorResponseFor(status int) []byte {
	if status == http.StatusNotFound {
		return []byte(prodNotFoundTmpl)
	}

	return []byte(prodErrorTmpl)
}```